test/JDBC/expected/getdate-vu-verify.out

```diff
@@ -77,3 +77,604 @@ int
 65536
 ~~END~~
 
+
+declare @x datetime2 = sysdatetime()
+select set_config('timezone', 'US/Pacific', false);
+declare @y datetime2 = sysdatetime()
+select set_config('timezone', 'UTC', false);
+select DATEDIFF(MINUTE, @y, @x)
+go
+~~START~~
+text
+US/Pacific
+~~END~~
+
+~~START~~
+text
+UTC
+~~END~~
+
+~~START~~
+int
+420
+~~END~~
+
+
+declare @x datetime2 = sysdatetimeoffset()
+select set_config('timezone', 'US/Pacific', false);
+declare @y datetime2 = sysdatetimeoffset()
+select set_config('timezone', 'UTC', false);
+select DATEDIFF(MINUTE, @y, @x)
+go
+~~START~~
+text
+US/Pacific
+~~END~~
+
+~~START~~
+text
+UTC
+~~END~~
+
+~~START~~
+int
+420
+~~END~~
+
+
+declare @x datetime2 = getdate()
+select set_config('timezone', 'US/Pacific', false);
+declare @y datetime2 = getdate()
+select set_config('timezone', 'UTC', false);
+select DATEDIFF(MINUTE, @y, @x)
+go
+~~START~~
+text
+US/Pacific
+~~END~~
+
+~~START~~
+text
+UTC
+~~END~~
+
+~~START~~
+int
+420
+~~END~~
+
+
+declare @x datetime2 = CURRENT_TIMESTAMP
+select set_config('timezone', 'US/Pacific', false);
+declare @y datetime2 = CURRENT_TIMESTAMP
+select set_config('timezone', 'UTC', false);
+select DATEDIFF(MINUTE, @y, @x)
+go
+~~START~~
+text
+US/Pacific
+~~END~~
+
+~~START~~
+text
+UTC
+~~END~~
+
+~~START~~
+int
+420
+~~END~~
+
+
+declare @x datetime2 = sysdatetime() AT TIME ZONE 'Pacific Standard Time'
+declare @y datetime2 = sysdatetime() AT TIME ZONE 'UTC'
+select DATEDIFF(MINUTE, @y, @x)
+go
+~~START~~
+int
+0
+~~END~~
+
+
+declare @x datetime2 = sysdatetimeoffset() AT TIME ZONE 'Pacific Standard Time'
+declare @y datetime2 = sysdatetimeoffset() AT TIME ZONE 'UTC'
+select DATEDIFF(MINUTE, @y, @x)
+go
+~~START~~
+int
+-420
+~~END~~
+
+
+declare @x datetime2 = getdate() AT TIME ZONE 'Pacific Standard Time'
+declare @y datetime2 = getdate() AT TIME ZONE 'UTC'
+select DATEDIFF(MINUTE, @y, @x)
+go
+~~START~~
+int
+0
+~~END~~
```

Comment:
What does this code block test ?

Reply:
Both @x and @y will contain GETDATE() but adjusted to different time zones. However, this doesn't affect the DATE component. It will store the same time, just tagged with different time zones, so the DATEDIFF(MINUTE, @y, @x) will calculate the difference in minutes, but since the timestamps are essentially still the same, the difference is 0. So, we are just testing the integrity of timestamps here.